### DISCUSSION TITLE
Appears to not be sending my questions

### DISCUSSION THREAD

**Discussion by ddmeunier** [jas3333/GPT-Helper#3] [Q&A]
Hi, First off, thanks for your work on this. I'm really excited to give it a go!

I have been able to install it successfully and it is running successfully. However, when I type in a question and hit <Enter> nothing happens. I have named my .env file GPT-Helper.env and put my Secret Key in there as REACT_APP_OPENAI_KEY=<my key>. I *think* I did that correctly? 

Any help to get the app to respond to my questions would be much appreciated. 

Thanks!
David

**Comment by johnbulliner** [comment]
I figured it out with the help of chatgpt. Do not name the file anything just name it '.env' with nothing in front. here is part of what it said: "What do i name the file before '.env'?
A: The file that you create in the client directory to store your OpenAI API key should be named .env. This is the default name that the GPT-Helper uses to look for the API key.

You should not include any other characters or numbers before the .env file extension. For example, the file should not be named mykey.env or openai.env. It should simply be named .env.

Once you have created the .env file and included your API key, you can start the GPT-Helper by running the npm start command in the client directory. This will start the GPT-Helper in your default web browser, and you should be able to use it to generate text.

If you are still having trouble getting the GPT-Helper to work, you can try restarting the server by running the npm stop command followed by the npm start command in the client directory. This will restart the server and may help resolve any issues you are experiencing.

You can also try clearing the cache and restarting your web browser. This can sometimes help resolve issues with web applications. To do this, close all open tabs in your web browser, and then restart the browser. Once the browser has restarted, navigate to the GPT-Helper and try using it again."

**Comment by jas3333** [comment]
Sorry, I didn't see your question until now, but looks like John gave the correct answer. It should just be named `.env`

**Reply by ddmeunier** [reply]
Thanks for the help. Unfortunately, I'm still having the same experience. After I type in a prompt, and hit enter, nothing happens. It's as if my <Enter> key isn't sending the command to the text box. :-( If you have any suggestions as to what I may have done wrong, I'd welcome them. Thanks guys!

**Reply by jas3333** [reply]
Open up the dev tools in the browser, click on console and it should display any errors.
